bundlegen/core/bundle_processor.py

```diff
@@ -193,6 +205,9 @@ def _process_process(self):
         self._add_bind_mount(
             '/usr/libexec/DobbyInit', '/usr/libexec/DobbyInit')
 
+        if self.createmountpoints:
```

Comment:
To me, it would make more sense to extend the `_add_bind_mount(...)` method to check the `createmountpoints` flag and create the blank file in rootfs there. Whilst that function isn't used too much at the moment, it means you can't forget to create the blank file if you do use it.

You might need to add an option in add_bind_mount to determine if it's a file or directory

Reply:
done